src/lib/DaprClient.php

```diff
@@ -202,25 +225,15 @@ public function delete(string $url, ?array $params = []): DaprResponse
         return $response;
     }
 
-    /**
-     * Shutdown the sidecar.
-     *
-     * @return void
-     * @param array $metadata Metadata to pass to the shutdown endpoint
-     *
-     * @throws DaprException
-     */
-    public function shutdown(array $metadata = []): void
+    private function set_current_reentrancy_id(array $headers): array
     {
-        $this->post("/shutdown", $metadata);
+        $current_id = $this->get_current_reentrancy_id();
+
+        return array_merge($headers, ['Dapr-Reentrancy-Id: '.$current_id], $this->extra_headers);
```

Comment:
If the header already exists, will this create duplicates?

Reply:
Good catch!